data/src/main/java/ca/bc/gov/data/datasource/remote/CommentRemoteDataSource.kt

```diff
@@ -33,4 +34,30 @@ class CommentRemoteDataSource @Inject constructor(
         }
         return response.toDto()
     }
+
+    suspend fun addComment(
+        parentEntryId: String?,
+        userProfileId: String?,
+        comment: String,
+        hdid: String,
+        accessToken: String
+    ): CommentDto {
+        val commentRequest = CommentRequest(
+            text = comment,
+            parentEntryId = parentEntryId,
+            userProfileId = userProfileId,
+            entryTypeCode = "Med"
```

Comment:
We need to pass entryTypeCode from the UI layer as this code will differ for lab tests.